frontend/src/hooks/useShortAddress.ts

```diff
@@ -0,0 +1,14 @@
+export const useShortAddress = () => {
```

Comment:
@denishacquin, where do we decide to create a new hook just to shorten an address, or do we decide to include this method in the wallet hook?

Reply:
was thinking the same, will bundle them together